content/events/workshop-amsterdam-2020-03-30/index.md

```diff
@@ -25,8 +25,9 @@ event:
     time: "11:30AM - 4:00pm (CET)"
     # The event description shown on the event list page.
     description: "Join us as we walkthrough Infrastructure as Code concepts via a series of hands-on labs. Topics covered include IaC fundamentals, in addition to application architectures and how to use IaC to create, update, and manage them."
-    # The calendly registrtion url for event specific pages.
-    calendly_url: "https://calendly.com/pulumi/kubecon?month=2020-03"
+    # The eventbrite registrtion url for event specific pages.
```

Comment:
```suggestion
    # The Eventbrite registration URL for the event.
```

Also, should we add this to the archetype? Should it _replace_ Calendly?
Should we add this to the archetype as well, as an option? Are we _switching_ to using EventBrite.

Reply:
So it is unclear if we are switching to Calendly. Eventbrite is being used in this specific case because we needed to be able to use promo codes. For now, this is a one-off.